src/main/resources/settings/elevator.yml

```diff
@@ -0,0 +1,5 @@
+range:
+  first: 5
+  second: 20
+  # From bedrock to max build height
+  third: 384
```

Comment:
If you look at the fluid voider for example, we split up settings for each individual item into their own files. It would also serve to remove the `getRange` function since you could just do `int range = getSetting("range")`